app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/UploadSourceChoiceBottomSheet.kt

```diff
@@ -40,41 +40,41 @@ fun UploadSourceChoiceBottomSheet(
     onFilePickerClicked: () -> Unit,
     closeBottomSheet: () -> Unit,
 ) {
-    if (isBottomSheetVisible()) {
-        SwissTransferBottomSheet(
-            onDismissRequest = closeBottomSheet,
-            titleRes = R.string.transferUploadSourceChoiceTitle,
-            content = {
```

Comment:
I think that one as well might have been better with the name

Reply:
I disagree for this one.

Reply:
I agree that this is still the most debatable one out there